.travis.yml

```diff
@@ -1 +1,18 @@
-script: exit 0
+language: node_js
+
+node_js:
+- '8.4.0'
+
+addons:
+  chrome: stable
+  apt:
+    packages:
+      - oracle-java8-set-default
```

Comment:
Added java package as nightwatch tests used in okta-vue library needs java, apparently.
The tests are still failing on travis. Investigating...